tsconfig.json

```diff
@@ -13,6 +13,7 @@
     "emitDecoratorMetadata": true,
     "declarationDir": "dist/types",
     "outDir": "dist",
+    "skipLibCheck": true,
```

Comment:
This is a "dirty hack" to get this Pull Request to compile. I'm not sure why this change will compile in `v12.17.0` but not in `v10.20.1` without telling typescript to skip checking libs in `node_modules`